backend/.env.example

```diff
@@ -1,6 +1,10 @@
-APP_ENV=dev
-APP_PORT=3008
-AUTH_SECRET=auth_secret_key
-
-
-DATABASE_URL="file:./somename.db"
\ No newline at end of file
+APP_ENV=dev
+APP_PORT=3008
+AUTH_SECRET=auth_secret_key
+
+
+DATABASE_URL="file:./somename.db"
+
+CLIENT_ID = 133300448301-7jr985i9aljbqdbf27okphk58b2f987n.apps.googleusercontent.com
+CLIENT_SECRET = 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

```

Comment:
```
CLIENT_ID = string
CLIENT_SECRET = string
```

We dont want to share private data to every user on github who has access to this repo)